pkg/tasks/certs.go

```diff
@@ -235,26 +234,26 @@ func approvePendingCSR(s *state.State, node *kubeoneapi.HostConfig, conn ssh.Con
 
 func validateCSR(spec certificatesv1.CertificateSigningRequestSpec, node *kubeoneapi.HostConfig) error {
 	if fmt.Sprintf("%s:%s", nodeUser, node.Hostname) != spec.Username {
-		return errors.New("")
+		return fmt.Errorf("CSR username %q and node hostname %q do not match", spec.Username, node.Hostname)
```

Comment:
I guess this should be `errors.New` instead.

Reply:
I like Errorf better since it manifests that "format" is used

Reply:
There’s also Errorf in the errors package, so we get to use same package everywhere. Just a nit though.